atom/browser/osr/osr_output_device.cc

```diff
@@ -84,12 +84,16 @@ void OffScreenOutputDevice::SetActive(bool active, bool paint) {
     return;
   active_ = active;
 
-  if (active_ && paint)
+  if (!active_ && !pending_damage_rect_.IsEmpty() && paint)
```

Comment:
@gerhardberger, @brenca 
Could you please check the correctness of how a merge conflict was resolved in this line?

The conflict was caused by these two commits:
by @gerhardberger https://github.com/electron/electron/commit/cc808e6cd23538902c989fcc77c2adc2fcea661a#diff-7add2caa56b7949489ffa7b88094c335
by @brenca https://github.com/electron/electron/pull/10510/files#diff-7add2caa56b7949489ffa7b88094c335